backend/src/controllers/userController.ts

```diff
@@ -103,15 +103,41 @@ export const updateUser = async (req: Request, res: Response) => {
   }
 };
 export const getAllUsers = async (req: Request, res: Response) => {
-  const { page, limit } = req.query;
+  const { page, limit, query } = req.query;
+  const { ownerAddress } = req.body
```

Comment:
Hey, its supposed to be `const ownerAddress = req.body.address`
Look at the middleware below. JWT is signed with the address. jwt.verify() returns the address back to the payload which is then set to req.body so that we can access in any controller function
![image](https://github.com/globe-and-citizen/cnc-portal/assets/31665486/c6207d0c-6204-4152-877d-a8fb3d0e1170)

Reply:
@farrellh1 filter need to have validation. And be well documented. 
If you can filter by anything you want, i don't think it's welcome

Reply:
it should be filtered by a key is that what you mean @hermannleboss ?

for example query params address will filter only for address, query params name will filter only by name